[2023-12-18T17:15:35.265+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: download_dashboard_automated_send_email.download_dashboard_automated_send_email scheduled__2023-12-17T17:15:13.761521+00:00 [queued]>
[2023-12-18T17:15:35.278+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: download_dashboard_automated_send_email.download_dashboard_automated_send_email scheduled__2023-12-17T17:15:13.761521+00:00 [queued]>
[2023-12-18T17:15:35.279+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-18T17:15:35.297+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): download_dashboard_automated_send_email> on 2023-12-17 17:15:13.761521+00:00
[2023-12-18T17:15:35.302+0000] {standard_task_runner.py:57} INFO - Started process 1083 to run task
[2023-12-18T17:15:35.306+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'download_dashboard_automated_send_email', 'download_dashboard_automated_send_email', 'scheduled__2023-12-17T17:15:13.761521+00:00', '--job-id', '235', '--raw', '--subdir', 'DAGS_FOLDER/download_dashboard_automated_send_email.py', '--cfg-path', '/tmp/tmpya3i756e']
[2023-12-18T17:15:35.308+0000] {standard_task_runner.py:85} INFO - Job 235: Subtask download_dashboard_automated_send_email
[2023-12-18T17:15:35.352+0000] {task_command.py:416} INFO - Running <TaskInstance: download_dashboard_automated_send_email.download_dashboard_automated_send_email scheduled__2023-12-17T17:15:13.761521+00:00 [running]> on host 83c86e565932
[2023-12-18T17:15:35.408+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='christinelee1114@gmail.com' AIRFLOW_CTX_DAG_OWNER='xin' AIRFLOW_CTX_DAG_ID='download_dashboard_automated_send_email' AIRFLOW_CTX_TASK_ID='download_dashboard_automated_send_email' AIRFLOW_CTX_EXECUTION_DATE='2023-12-17T17:15:13.761521+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-17T17:15:13.761521+00:00'
[2023-12-18T17:16:07.120+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/download_dashboard_automated_send_email.py", line 53, in download_dashboard_automated_send_email
    driver = webdriver.Chrome(options=chrome_options)
  File "/home/airflow/.local/lib/python3.8/site-packages/selenium/webdriver/chrome/webdriver.py", line 45, in __init__
    super().__init__(
  File "/home/airflow/.local/lib/python3.8/site-packages/selenium/webdriver/chromium/webdriver.py", line 50, in __init__
    self.service.start()
  File "/home/airflow/.local/lib/python3.8/site-packages/selenium/webdriver/common/service.py", line 102, in start
    self.assert_process_still_running()
  File "/home/airflow/.local/lib/python3.8/site-packages/selenium/webdriver/common/service.py", line 115, in assert_process_still_running
    raise WebDriverException(f"Service {self._path} unexpectedly exited. Status code was: {return_code}")
selenium.common.exceptions.WebDriverException: Message: Service /home/airflow/.cache/selenium/chromedriver/linux64/120.0.6099.71/chromedriver unexpectedly exited. Status code was: 127

[2023-12-18T17:16:07.134+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=download_dashboard_automated_send_email, task_id=download_dashboard_automated_send_email, execution_date=20231217T171513, start_date=20231218T171535, end_date=20231218T171607
[2023-12-18T17:16:07.146+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/email.py:154: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2023-12-18T17:16:07.147+0000] {email.py:270} INFO - Email alerting: attempt 1
[2023-12-18T17:16:08.023+0000] {email.py:281} INFO - Sent an alert email to ['christinelee1114@gmail.com']
[2023-12-18T17:16:09.216+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 235 for task download_dashboard_automated_send_email (Message: Service /home/airflow/.cache/selenium/chromedriver/linux64/120.0.6099.71/chromedriver unexpectedly exited. Status code was: 127
; 1083)
[2023-12-18T17:16:09.262+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-18T17:16:09.274+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
